WcaOnRails/app/helpers/notifications_helper.rb

```diff
@@ -67,6 +67,16 @@ def notifications_for_user(user)
       }
     end
 
+    user.delegated_competitions.visible.over
+      .where("start_date >= ?", DelegateReport::REPORTS_ENABLED_DATE) # Don't notify about very old competitions without reports.
```

Comment:
I wonder if filtering the old competitions out is a good idea:
:heavy_plus_sign: delegates don't end up with dozens of notifications that are likely to stay there for ages
:heavy_minus_sign: delegates aren't informed which of their old competitions are missing corresponding reports